specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/privateLinkServicesForExactDataMatchUpload.json

```diff
@@ -0,0 +1,1500 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "Resource"
+        ],
+        "description": "Update the metadata of a privateLinkServicesForExactDataMatchUpload instance.",
+        "operationId": "privateLinkServicesForExactDataMatchUpload_Update",
+        "x-ms-examples": {
+          "Patch service": {
+            "$ref": "./examples/EdmUploadServicePatch.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "name": "servicePatchDescription",
+            "in": "body",
+            "description": "The service instance metadata and security metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServicesPatchDescription"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The tags for the resource were updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/privateLinkServicesForExactDataMatchUploadDescription"
+            }
+          },
+          "202": {
+            "description": "The tags for the resource were updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/privateLinkServicesForExactDataMatchUploadDescription"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "delete": {
+        "tags": [
+          "Resource"
+        ],
+        "description": "Delete a service instance.",
+        "operationId": "Services_Delete",
+        "x-ms-examples": {
+          "Delete service": {
+            "$ref": "./examples/EdmUploadServiceDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
>"200" [](start = 10, length = 5)

No need to return 200 if you are already returning 204, see [here](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#delete-resource-asynchronously)